.github/workflows/avm.res.desktop-virtualization.host-pool.yml

```diff
@@ -0,0 +1,83 @@
+name: "avm.res.desktop-virtualization.host-pool"
```

Comment:
Hi @moisesjgomez, also here, codeowners file should be updated. For guidelines on how to configure teams please refer to https://azure.github.io/Azure-Verified-Modules/specs/shared/#team-memberships

Reply:
Codeowners file update and teams created.